### ISSUE TITLE
Command line markdown import

### ISSUE THREAD

**Issue by AJigsawnHalo** [davidgiven/wordgrinder#280]
Importing Markdown files while wordgrinder is open works fine but using the `convert` option in the command line is still limited to the old allowed file types. Is this intended behavior?

**Comment by davidgiven** [comment]
No, it's not --- the list of import formats didn't get updated when it should have been. Will fix.
